### ISSUE TITLE
Android logo instead the new program icon

### ISSUE THREAD

**Issue by auge8472** [jonan/ForkHub#174]
After changing the program icon from the GitHub app to the new one the desktop screen shows the 3D-Android-logo. I would expect the new program logo as the desktop icon. Any ideas?

**Comment by jonan** [comment]
Have you tried restarting the device? Might be a cache issue.

**Comment by auge8472** [comment]
Yes I did it a few minutes ago. :-)

**Comment by jonan** [comment]
What Android version are you using?

**Comment by auge8472** [comment]
I use CM 11 (Android 4.4.4) on a Samsung Galaxy 2

**Comment by jonan** [comment]
Thanks, I'll investigate it.

**Comment by auge8472** [comment]
I found a very simple work around after I puzzled with restarts after the update that came up in the meantime. Deleting the link with the Android standard icon on the desktop and recreating it from the main menu, where the new icon was used since it's introduction, solved my problem.

I don't know, if the underlying issue is it worth to investigate it deeper, when a (simple) work around for this cosmetical issue exists. So it's up to you to close the issue or not.
